schemas/context/profile-contact-crm.schema.json

```diff
@@ -0,0 +1,154 @@
+          "title": "Email Bounced Date",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the date when the email was bounced.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:emailBouncedReason": {
+          "title": "Email Bounced Reason",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the reason for bounced email.",
+          "type": "string"
+        },
+        "xdm:emailBounced": {
+          "title": "Email Bounced Indicator",
+          "description": "If bounce management is activated and an email is sent to a contact, indicates whether the email bounced (true) or not (false).",
+          "type": "boolean"
+        },
+        "xdm:leadSource": {
+          "title": "Lead Source",
+          "description": "Field recording what source the lead originated from.",
+          "type": "string"
+        },
+        "xdm:contactDescription": {
+          "title": "Contact Description",
+          "description": "Contact description.",
+          "type": "string"
+        },
+        "xdm:contactStatus": {
+          "title": "Contact Status",
+          "description": "Current status of the contact.",
+          "type": "string"
+        },
+        "xdm:contactStatusDetail": {
+          "title": "Contact Status Detail",
+          "description": "Additional Details of the current status of the contact.",
+          "type": "string"
+        },
+        "xdm:contactType": {
+          "title": "Contact Type",
+          "description": "Type of the contact.",
+          "type": "string"
+        },
+        "xdm:userID": {
+          "title": "User Id",
+          "description": "User id.",
+          "type": "string"
+        },
+        "xdm:originalReferrer": {
+          "title": "Original Referrer",
+          "description": "The original referrer is typically the page that leads to a form fill out.",
+          "type": "string"
+        },
+        "xdm:originalSearchEngine": {
+          "title": "Original Search Engine",
+          "description": "Search engine that the original referrer used.",
+          "type": "string"
+        },
+        "xdm:originalSearchPhrase": {
+          "title": "Original Search Phrase",
+          "description": "Search phrase used by the original referrer.",
+          "type": "string"
+        },
+        "xdm:originalSourceInfo": {
+          "title": "Original Source Info",
+          "description": "Original source info.",
+          "type": "string"
+        },
+        "xdm:originalSourceType": {
+          "title": "Original Source Type",
+          "description": "Original ource type.",
+          "type": "string"
+        },
+        "xdm:timezone": {
+          "title": "Timezone",
+          "description": "Timezone of the contact's location.",
+          "type": "string"
+        },
+        "xdm:emailOptOutReason": {
```

Comment:
create an Opt-Out data type with the properties as "optOut","optOutDate" and "optOutReason" and then use it against different concepts like Email, Phone, Fax etc.

Reply:
There is already an existing data type "optinout" which has optOut options for different channels like email, phone fax etc. If the other optOut properties need to be in a mixin/data type then we should look at extending the existing mixin rather than creating a new one. I will see if I can suggest something and share with the team soon.

Reply:
That Optout is for Privacy signals, this is optOut for any given subject in the landscape of not receiving notifications

Reply:
The "optinout.schema.json" is not explicitly defined for Privacy signals. It just describes user communication preferences by channel. The profile-privacy is the privacy specific mixin. That answers your concern. I would still suggest to extend the "optinout.schema.json" data type to add the additional properties.